src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -18,58 +29,66 @@
   }
 }
 
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
+.nav-dropdown {
   position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
+  z-index: 1;
+  top: 0;
   height: auto;
-  max-height: 75vh;
-  overflow-y: auto;
-
-  padding: 20px;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.5);
-  border-radius: 0 0 24px 24px;
-  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  width: $search-results-width;
+  margin-top: 66px;
+  width: 290px;
   background-color: $saturated-black;
+  text-align: left;
+  box-shadow: 0 10px 30px 0 rgba($black, 0.5);
+  border-radius: 0 0 $border-radius--input $border-radius--input;
+  transform: translateY(-100px);
+  transform-origin: top right;
+  transition: all 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  pointer-events: none;
 
-  .nav-search-result-number {
+  &.show {
+    transform: scale(1) translateY(0px);
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  .nav-dropdown-title {
+    padding: 0 12px;
     opacity: 0.5;
     font-size: 0.9rem;
+    b {
+      font-weight: 500;
+    }
   }
 
-  .row {
-    padding: 10px;
-    flex-direction: row;
+  &.nav-dropdown--search {
```

Comment:
This syntax (`.&`) will combine this class with the parent class, which doesn't really align to how we tend to use BEM.

What's the intent of this particular section, and can we do it with our standard BEM patterns?